src/hex/types.ts

```diff
@@ -6,10 +6,12 @@ export interface Point {
   y: number
 }
 
-export enum Orientation {
-  FLAT = 'FLAT',
-  POINTY = 'POINTY',
-}
+export const Orientations = {
```

Comment:
For what it's worth, I think enums should be [singular](https://stackoverflow.com/a/15756009/660260).

Reply:
Yeah, it makes sense. I wanted to keep `Orientation` constant so that I don't have to rename that throughout the project as well.